README.md

```diff
@@ -33,7 +33,7 @@ When running with `parallel` set to true, the final reports can be merged by usi
 
 ## Configuration
 
-Configuration is optional. It should be put in a file at `config/coverage.js`. 
+Configuration is optional. It should be put in a file at named `coverage.js` inside your ember app config folder (defaults to `config/`).
```

Comment:
I'd prefer to keep this README line unchanged. 99% of our users will be using `config/coverage.js`, and I think rewording like jisnjust makes it hard to grok for the vast majority use case. I think a parenthetical aside or a second sentence mentioning that we honor `configPath` is enough...